prisma/schema.prisma

```diff
@@ -12,88 +12,153 @@ datasource db {
 
 // Here goes models
 model UserMetadata {
-  id            Int      @id @default(autoincrement())
-  userId        Int      @unique()
-  teamId        Int
-  isNewEmployee Boolean
-  createdAt     DateTime @default(now())
-  updatedAt     DateTime @updatedAt
+  id                         Int               @unique() @default(autoincrement())
+  userId                     String            @unique() @db.Uuid
+  userName                   String            @unique()
+  profilePicture             String?
+  isNewEmployee              Boolean
+  designation                String?
+  dateOfJoining              DateTime
```

Comment:
is dateOfJoining same as createdAt?

Reply:
No, its the date at which an employee joined in the organization.